File: services/reactor/core/src/main/java/com/torin/core/dto/ParticipantChangedDto.java
package com.torin.core.dto;

import java.time.Instant;

import com.fasterxml.jackson.annotation.JsonProperty;

public record ParticipantChangedDto(Long id, @JsonProperty("participants_count") Long participantsCount,
        @JsonProperty("id_group") Long idGroup, Instant date) {
}

File: services/telegramservice/DTOs/DTO/ParticipantChangedDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace DTOs.DTO
{
    public class ParticipantChangedDTO
    {
        public long IdGroup { get; set; }
        public long ParticipantsCount { get; set; }
        public DateTime Date { get; set; }
    }
}